FreestarAds-Ogury-PreRelease.podspec

```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |spec|
+  spec.name                = "FreestarAds-Ogury-PreRelease"
+  spec.version             = "1.0.0.1-beta-1"
```

Comment:
Partner adapter semantic versioning schemes should not start at 1.0.0...

Reply:
As per @kkawai comment this should be `2.1.0.0` right ?